app/components/ui/set-up-domain/progress-header/index.js

```diff
@@ -0,0 +1,35 @@
+// External dependencies
+import i18n from 'i18n-calypso';
+import React, { PropTypes } from 'react';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+
+// Internal dependencies
+import ProgressBar from 'components/ui/progress-bar';
+import styles from './styles.scss';
+
+const ProgressHeader = ( { children, handleOnExitClick, progress } ) => {
+	return <div className={ styles.headerContainer }>
+		<div className={ styles.header }>
+			<h1 className={ styles.headerText }>
+				<span className={ styles.setUpLabel }>
+					{ i18n.translate( 'Setup: ' ) }
+				</span>
+				{ children }
+			</h1>
+			<div className={ styles.progressHeader }>
+				<div className={ styles.fullWidth }>
+					<ProgressBar progress={ progress } />
+				</div>
+				<span onClick={ handleOnExitClick } className={ styles.exit }>Exit</span>
```

Comment:
Instead of making this a `span` with an `onClick` handler, what do you think about using an anchor tag?

```js
<a href={ getPath( 'myDomains' ) }>
```

This provides a slightly better UX (the user can see where the link points and see the context menu that appears for a link in the browser) and saves us from adding a container for this component.